pbft/pbft.cpp

```diff
@@ -1394,17 +1438,80 @@ pbft::save_checkpoint(const pbft_msg& msg)
     }
 }
 
+
+std::map<bzn::hash_t, int>
```

Comment:
May be cleaner to return a map of hashes to references (or std::reference_wrappers, as the case may be...)